_posts/2023-07-07-newpipe-net-dmca-google-search.md

```diff
@@ -0,0 +1,20 @@
+---
+layout: post
+title: "newpipe.net removed from Google search results due to DMCA take down request"
+short: "newpipe.net removed from Google search results"
+date: 2023-07-07 15:00:00 + 02:00
+categories: [pinned, announcement]
+author: <a href="https://github.com/theScrabi">@theScrabi</a>
+image: newpipe
+excerpt_separator: <!-- more -->
+---
+Dear NewPipe community and other beloved creatures,
+
+Maybe you have already noticed this: if you search for "newpipe" via Google, you will find everything about us, except the official website. This is because Google accepted a DMCA request from the French company "Because Music" against our homepage, which means that from now on the landing page at newpipe.net will not show up in Google search results. You can find the full DMCA [here in the Lumen database](https://lumendatabase.org/notices/34149383).  
+
+While this is not the worst thing that could have happened, it is still rather nasty. Because the official website is hidden in the search results, fake clones of the site will tend to rank higher in the results, and our site may lose relevance. We are currently discussing whether to take legal action against this DMCA.
```

Comment:
```suggestion
While worse things could have happened as the result of such a takedown request, Google's unlisting of our homepage creates a variety of problems. Given the fact that Google no longer counts visits to our site, it lose relevance. As a result, websites for fake clones of NewPipe that mimic our homepage will tend to rank higher in the results. This could ultimately cause users to fall for such scams.
```